### PR TITLE
docs: update nginx config adding breaking changes

### PR DESCRIPTION
**Pull request by fzipi**
Signed-off-by: Felipe Zipitria <felipe.zipitria@owasp.org>

Update documentation on beaking changes for nginx config.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +135,9 @@ $ docker run -p 8080:80 -e SERVER_NAME=myhost my-modsec
 | SSL_CIPHER_SUITE | A string indicating the cipher suite to use. Uses OpenSSL [list of cipher suites](https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_ciphersuites.html) (Default: `"ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384"` |
 | SSL_ENGINE  | A string indicating the SSL Engine Operation Switch (Default: `off`) | 
 | SSL_HONOR_CIPHER_ORDER | A string indicating if the server should [honor the cipher list provided by the client](https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslhonorcipherorder) (Allowed values: `on`, `off`. Default: `off`) |
+    # If you need to modify nginx config, mount your volume as the template file
+    # volumes:
+    #  - ./nginx/nginx.conf:/etc/nginx/templates/nginx.conf.template:rw
```
Comment (theseion) [inline README.md:140]: This is in the wrong place, isn't it?

**Review by theseion — CHANGES_REQUESTED** [review]
Good idea. Looks good, except for the misplaced (?) comment.

File: README.md (hunk 1 above)
Comment (fzipi360) [reply README.md]: Definitely. Bad paste :)

Comment (fzipi) [reply README.md]: Yes, bad paste.

File: README.md [hunk 2]
```diff
@@ -66,6 +66,19 @@ This image only contains ModSecurity built from the code provided on the [ModSec
...
+
+| ⚠️ WARNING          |
+|:---------------------------|
+| Nginx based images are now based on upstream nginx. This changed the way the config file for nginx is generated.  |
+
+If using the [Nginx environment variables](https://github.com/coreruleset/modsecurity-docker#nginx-env-variables) is not enough for your use case, you can mount your own `nginx.conf` file as the new template for generating the base config.
+
+An example can be seen in the [docker-compose](https://github.com/coreruleset/modsecurity-docker/blob/master/docker-compose.yml) file.
+
+> 💬 What happens if I want to make changes in a different file, like `/etc/nging/conf.d/default.conf`?
```
Comment (theseion) [inline README.md:79]:
```suggestion
> 💬 What happens if I want to make changes in a different file, like `/etc/nginx/conf.d/default.conf`?
```

File: README.md [hunk 3]
```diff
@@ -66,6 +66,19 @@ This image only contains ModSecurity built from the code provided on the [ModSec
...
+> You mount your local file, e.g. `nginx/default.conf` as the new template: `/etc/nginx/templates/conf.d/default.conf.template`. You can do the similar to other files. Files in the templates directory will be copied preserving the subdirectories structure.
```
Comment (theseion) [inline README.md:80]:
```suggestion
> You mount your local file, e.g. `nginx/default.conf` as the new template: `/etc/nginx/templates/conf.d/default.conf.template`. You can do this similarly with other files. Files in the templates directory will be copied and subdirectories will be preserved.
```

**Review by theseion — CHANGES_REQUESTED** [review]
A typo and some wording changes.

**Review by theseion — APPROVED** [review]

**Pull request merged**